src/js/utils/Util.js

```diff
@@ -92,37 +92,66 @@ const Util = {
   },
 
   /**
+   * Underscore.js 1.8.3. For more information,
+   * see http://underscorejs.org/docs/underscore.html#section-82
    * @param {Function} func A callback function to be called
    * @param {Number} wait How long to wait
-   * @returns {Function} A function, that, after triggered the first time, will
-   * wait for a period of time before the next trigger. If triggered during the
-   * wait, the function will be invoked immediately after the wait is over.
-   * This function is specifically made for React events, hence the nativeEvent
-   * lookup.
+   * @param {Object} [options]
+   * @param {Object} [options.leading = false]
```

Comment:
Could we add some comments about what `leading` and `trailing` options are? Based on the description below in `@returns` I'm still not sure what they do